### ISSUE TITLE
Is this still active?

### ISSUE THREAD

**Issue by kechiao** [alex-ong/DDRPad#1]
Hi there Alex,

I came across this repo and was curious about the results of the pad. I cloned it and saw you were playing Max 300 but I was wondering if there were any issues / problems with the pad? I might be taking some inspiration from your v2 design, making the arrow panels with 1cm polycarbonate and using copper tape as the method of wiring.

Thanks!

**Comment by alex-ong** [comment]
Hi Kechiao,

Definitely use it as reference material, with the caveat that v2 is completely untested. 
V1 works but is obviously less aesthetically pleasing. I'll upload a gif of that.

In terms of maintenance, i probably maintain it after every 3-5 hours of play. Usually its just the gaff holding the 3mm ply down is coming loose. I think using a T-nut or so would help but since it already works, and a 5-10 minute check once a week seems fine i've left it.

The code for the pro-micro is pretty good; you can adjust sensitivity and debounce per panel. In particular, down arrow needs to be less sensitive coz you use heels and up arrow more sensitive since its toes only.

There's quite a few designs on reddit so i suggest you look there too.

Total BOM was around $147 AUD. If you build two its much cheaper.
I think i spent around $150 but wasted a lot of $$$ on velostat (bought 5 sheets). I had lots of the components (Gaffer Tape, ethernet cable, Arduino, PVA glue etc etc)

I've updated v1 folder: https://github.com/alex-ong/DDRPad/tree/master/photos/v1/design
and the repo readme with some videos:
https://github.com/alex-ong/DDRPad

**Comment by kechiao** [comment]
Hi Alex,

Awesome, thanks so much for updating the repo. I've been eyeing your design and Promit's DanceForce designs. I really like the separate panel design you have.

One last question: in the v2 design, it seems like both the left and up arrow channels also have down, right (respectively) inputs tied to that panel. Would this not cause double presses, or do you only put copper/conductive material covering the velostat only? 

Thanks again!

**Comment by alex-ong** [comment]
> or do you only put copper/conductive material covering the velostat only?

That's right! you only need copper/velostat on 2/3rds of the arrow for it to be effective.  so for those arrow channels, i would just carefully place the copper so that there's no overlap.

I've now added a GUI too so that my firmware can change sensitivity without have to recompile all the code.

**Comment by alex-ong** [comment]
for future refernce, Velostat is a scam now - closing and please use `teejusb fsr` https://github.com/teejusb/fsr
